label_studio/ml_models/models.py

```diff
@@ -159,3 +172,27 @@ def output_file_name(self):
     @property
     def error_file_name(self):
         return f'{self.project.id}_{self.model_version.pk}_{self.pk}/error.csv'
+
+
+@receiver(pre_delete, sender=ModelVersion)
```

Comment:
Consider switching from using a signal to putting this inside `ModelVersion.delete()`: https://www.notion.so/humansignal/Backend-Handbook-ce7324a497cb4b40b91911a429e34fcb?pvs=4#052c958f2f9c48b6b6f1b207bca3674d